aea/protocols/generator.py

```diff
@@ -343,15 +321,18 @@ def _message_class_str(self) -> str:
 
         cls_str += '            # Light Protocol 2\n'
         cls_str += '            # Check correct performative\n'
-        cls_str += '            assert performative in self.performatives, \"performative is not in the list of allowed performative\"\n\n'
+        cls_str += '            assert performative in self.performatives,' \
+                   ' \"performative is not in the list of allowed performative\"\n\n'
 
         cls_str += '            # Check correct contents\n'
         cls_str += '            content_sequence_definition = self.speech_acts[performative]  # type is List\n'
         cls_str += '            # Check number of contents\n'
-        cls_str += '            assert len(contents) == len(content_sequence_definition), \"incorrect number of contents\"\n'
+        cls_str += '            assert len(contents) == len(content_sequence_definition), ' \
+                   '\"incorrect number of contents\"\n'
         cls_str += '            # Check the content is of the correct type\n'
         cls_str += '            for x in range(len(content_sequence_definition)):\n'
```

Comment:
x is not a permitted variable name according to our style guide. Please use a descriptive variable name

Reply:
Done